.travis.yml

```diff
@@ -6,9 +6,9 @@ cache:
 notifications:
   email: false
 node_js:
-  - '8'
   - '10'
   - '12'
+  - '13'
```

Comment:
The Node.js folks don't recommend v13 for production since it wont be promoted to LTS. Maybe both just v10 and v12 will suffice?

Reply:
It's the testing environment not the production environment. If this adds too much work we can remove it. Otherwise it seems useful to monitor support.